content/blog/2022/10/envvar-policy.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Enforcing compliance in your environment variables
+authors:
+- José Guilherme Vanz
+date: 2022-11-01
+---
+
+We're glad to present to Kubewarden users the new [environment-variable-policy](https://github.com/kubewarden/environment-variable-policy). It's able to inspect containers, init containers, and ephemeral containers, and restricts their usage of environment variables by checking their names and values defined under the containers' [`env[*]`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#envvar-v1-core).
+
+As always, the policy can be found in [Github](https://github.com/kubewarden/environment-variable-policy) and all the  artifacts, including the BOM files, are signed with Sigstore.
+
+## What is so useful about the new environment-variable policy?
+
+This policy complements the recently released  [env-variable-secrets-scanner](https://www.kubewarden.io/blog/2022/10/env-var-secrets/) policy. Both policies focus on validating the environment values provided to Kubernetes Pod objects. In this new policy, users can validate which variables name and values their resources should have. For that, the policy gives four rule operators to control the environment variable defined in the containers. They are:
+
+### anyIn
+
+`anyIn` works as a deny list of your usage of volumes. It checks if any of the `environmentVariables` are in the Pod/Workload resource:
+```yaml
+settings:
+  rules:
+    - reject: anyIn
+      environmentVariables:
+        - name: "envvar1"
+        - name: "envvar2"
+```
+
+### anyNotIn
+
+`anyNotIn` works as an allow list. Checks if any of the `environmentVariables` are not in the Pod/Workload resource:
+
+```yaml
+settings:
+  rules:
+    - reject: anyNotIn
+      environmentVariables:
+        - name: "envvar2"
+          value: "envvar2_value"
+        - name: "envvar3"
+```
+
+### allAreUsed
+
+`allAreUsed` the container cannot use all listed volumes at once. Checks if all of the `environmentVariables` are in the Pod/Workload resource:
+
+```yaml
+settings:
+  rules:
+    - reject: allAreUsed
+      environmentVariables:
+        - name: "envvar2"
+          value: ""
+```
+
+### notAllAreUsed
+
+`notAllAreUsed` the container can use all listed volumes at once, but only one of them. Checks if all of the `environmentVariables` are not in the Pod/Workload resource
+
+```yaml
+settings:
+  rules:
+    - reject: notAllAreUsed
+      environmentVariables:
+        - name: "envvar3"
+          value: "envvar3_value"
+        - name: "envvar4"
+          value: "envvar4_value"
+```
+
+
+The user is allowed to define multiple rules in the policy settings. The resource must be approved by all of them in order to be accepted into the cluster.
```

Comment:
```suggestion
## Using multiple rules

The user is allowed to define multiple rules in the policy settings. The resource must be approved by all of them in order to be accepted into the cluster.
```

This is worth to be highlighted inside of a dedicated chapter, otherwise it looks like this information is specific to the `notAllAreUsed` rule.